### PR TITLE
Never use /api/v1 from Gitea UI Pages

### PR DESCRIPTION
**Pull request by lunny**
Reusing `/api/v1` from Gitea UI Pages have pros and cons.
Pros:
1) Less code copy

Cons:
1) API/v1 have to support shared session with page requests.
2) You need to consider for each other when you want to change something about api/v1 or page.

This PR moves all dependencies to API/v1 from UI Pages.

Partially replace #16052

### CODE REVIEW

**Comment by wxiaoguang** [comment]
If the handlers for `/api/ui/` is the same as `/api/v1`, can these `ui` handlers just proxy/pass the request to the `v1` handlers? IIRC the newly added code are copied from the v1 handlers.

**Comment by lunny** [comment]
> If the handlers for `/api/ui/` are the same as `/api/v1`, can these `ui` handlers just proxy/pass the request to the `v1` handlers? IIRC the newly added code are copied from the v1 handlers.

They cannot be reused simply. We need many work to do like what I tried in #16052. Currently they are just copied but I think they need to be changed in coming PRs.

**Review by wxiaoguang — APPROVED** [review]
If copying code here is the only choice, then it's fine.

Another question is can these code be covered by tests?

**Comment by 6543** [comment]
@lunny you missed one:

`repo/issue/view_content/sidebar.tmpl` at line 432

**Comment by lunny** [comment]
> 

Thanks, will fix that later.

File: integrations/org_test.go [hunk 1]
```diff
@@ -173,3 +175,30 @@ func TestOrgRestrictedUser(t *testing.T) {
...
+	assert.Len(t, results.Data, 1)
+	assert.Equal(t, "test_team", results.Data[0].Name)
+
+	// no access if not organization member
+	user5 := unittest.AssertExistsAndLoadBean(t, &user_model.User{ID: 5}).(*user_model.User)
+	session = loginUser(t, user5.Name)
+	csrf = GetCSRF(t, session, "/"+org.Name)
+	req = NewRequestf(t, "GET", "/org/%s/teams/-/search?q=%s", org.Name, "team")
+	req.Header.Add("X-Csrf-Token", csrf)
+	session.MakeRequest(t, req, http.StatusForbidden)
```
Comment (6543) [inline integrations/org_test.go]:
```suggestion
	session.MakeRequest(t, req, http.StatusNotFound)
```

**Review by 6543 — APPROVED** [review]

**Pull request merged**